### PR TITLE
Create pl.json

### PR DESCRIPTION
**Pull request by LukaszP2**
polish translation

### CODE REVIEW

File: custom_components/google_home/translations/pl.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Autoryzacja Google Home",
+        "description": "Jeśli potrzebujesz pomocy przejdź na: https://github.com/leikoilja/ha-google-home. Uzyj swoich danych konta Google. Bezpieczniej/łatwiej jest wygenerować hasło do aplikacji i użyć go zamiast rzeczywistego hasła. Nadal masz taki sam dostęp jak przy zwykłym haśle, ale jest to lepsze niż używanie prawdziwego hasła podczas pisania skryptów (https://myaccount.google.com/apppasswords). 
+        Jeśli nie chcesz, zwykłe hasło do konta Google powinno działać.",
```
Comment (KapJI) [inline custom_components/google_home/translations/pl.json:7]: JSON can't contain a new line.

**Review by KapJI — CHANGES_REQUESTED** [review]
Please update README.md.

There is also #563, please a look if some wording is better there.

File: custom_components/google_home/translations/pl.json (hunk 1 above)
Comment (LukaszP2) [reply custom_components/google_home/translations/pl.json]: It should works ;)

**Comment by KapJI** [comment]
No response from the author, closing in favour of #563.

**Pull request closed without merging**